samples/wifi/thread_coex/README.rst

```diff
@@ -0,0 +1,400 @@
+.. _wifi_thread_coex_sample:
+
+Wi-Fi: Thread coexistence
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Thread coexistence sample demonstrates coexistence between Wi-Fi® and OpenThread device radios in 2.4 GHz frequency.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Running coexistence sample test cases require additional software such as the Wi-Fi **iperf** application.
+When the sample runs Wi-Fi UDP throughput in client mode, a peer device (test PC) runs UDP throughput in server mode using the following command:
+
+.. code-block:: console
+
+   $ iperf -s -i 1 -u
+
```

Comment:
I think this info should remain where it was. Its just about the iperf application we needed to move.